tests/baselines/reference/alwaysStrictNoImplicitUseStrict.js

```diff
@@ -0,0 +1,17 @@
+//// [alwaysStrictNoImplicitUseStrict.ts]
+
+module M {
+    export function f() {
+        var arguments = [];
+    }
+}
+
+//// [alwaysStrictNoImplicitUseStrict.js]
+"use strict";

```

Comment:
Can you add a check in program.ts::verifyCompilerOptions to flag teh use of `--noImplicitUseStrict` and `--alwaysStrict` as invalid combination.. so somthing like
```ts
if (options.noImplicitUseStrict && options.alwaysStrict) {
    programDiagnostics.add(createCompilerDiagnostic(Diagnostics.Option_0_cannot_be_specified_with_option_1, "noImplicitUseStrict ", "alwaysStrict"));
}
```